### PR TITLE
Geniatech.eyeTV version 4

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106254)

### CODE REVIEW

**Comment by upintheairsheep** [comment]
The automatic installation of a dependency requires user attention, not sure if this should be a dependency or interactive only installer.

**Comment by Trenly** [comment]
> The automatic installation of a dependency requires user attention, not sure if this should be a dependency or interactive only installer.

It's both because there's a dependency on LavFilters, but also it's interactive only due to a driver install prompt

[Policy] Interactive Only
[Policy] Dependencies

File: manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Geniatech.eyeTV
+PackageVersion: 4
```
Comment (Trenly) [inline manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.installer.yaml:5]:
```suggestion
PackageVersion: 4.6.0.0
```
This should match control panel. The version folder will need to be renamed

File: manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Geniatech.eyeTV
+PackageVersion: 4
```
Comment (Trenly) [inline manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.6.0.0
```
This should match control panel. The version folder will need to be renamed

File: manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Geniatech.eyeTV
+PackageVersion: 4
+PackageLocale: en-US
+Publisher: Geniatech
```
Comment (Trenly) [inline manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.locale.en-US.yaml:7]:
```suggestion
Publisher: Christian Hackbart
```
This must match control panel

File: manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Geniatech.eyeTV
+PackageVersion: 4
```
Comment (Trenly) [inline manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.yaml:5]:
```suggestion
PackageVersion: 4.6.0.0
```
This should match control panel. The version folder will need to be renamed

File: manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Geniatech.eyeTV
+PackageVersion: 4
+PackageLocale: en-US
+Publisher: Geniatech
+PackageName: eyeTV 4
+License: PRoprietary
+ShortDescription: eyeTV software
```
Comment (Trenly) [inline manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.locale.en-US.yaml:10]: Please provide a better description of what eyeTV is used for

File: manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Geniatech.eyeTV
+PackageVersion: 4
+PackageLocale: en-US
+Publisher: Geniatech
+PackageName: eyeTV 4
+License: PRoprietary
```
Comment (Trenly) [inline manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.locale.en-US.yaml:9]:
```suggestion
License: Proprietary

```

File: manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Geniatech.eyeTV
+PackageVersion: 4
+PackageLocale: en-US
+Publisher: Geniatech
+PackageName: eyeTV 4
```
Comment (Trenly) [inline manifests/g/Geniatech/eyeTV/4/Geniatech.eyeTV.locale.en-US.yaml:8]:
```suggestion
PackageName: EyeTV
```
This should match control panel as closely as possible

**Review by Trenly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**